src/loader/class-post-loader.php

```diff
@@ -182,17 +189,41 @@ function ( array $postarr ) use ( $loader_settings ) {
 									return null;
 								}
 
+								$this->listen_for_attachment_creation();
+
+								// Ensure 'post_content' is a string. This is more than
+								// likely a "Block_Converter" instance. Converting is
```

Comment:
```suggestion
								// likely a "Block_Converter" instance. Converting this
```
This is my best guess of what you meant.